kafka_consumer/tests/python_client/test_integration.py

```diff
@@ -75,9 +75,12 @@ def test_monitor_broker_highwatermarks(dd_run_check, check, aggregator, is_enabl
 
     # After refactor and library migration, write unit tests to assert expected metric values
     aggregator.assert_metric('kafka.broker_offset', count=metric_count)
+    aggregator.assert_metric('kafka.consumer_offset', count=2)
+    aggregator.assert_metric('kafka.consumer_lag', count=2)
 
     for tag in topic_tags:
         aggregator.assert_metric_has_tag('kafka.broker_offset', tag, count=2)
 
     aggregator.assert_metric_has_tag_prefix('kafka.broker_offset', 'partition', count=metric_count)
+    aggregator.assert_all_metrics_covered()
```

Comment:
Small nit: it looks like this test is specifically testing broker offset currently. Could you make your changes in a new test or rename this test to be more encompassing of the new asserts?

Reply:
You're right! Actually this is already in other tests https://github.com/DataDog/integrations-core/blob/c122a420a34b6cc8625f2de9603260a82c9abd86/kafka_consumer/tests/common.py#L26 so I'm just gonna drop this 🙂